internal/syncd/integrations/ecr/sqsmock.go

```diff
@@ -0,0 +1,43 @@
+package ecr
+
+import (
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/stretchr/testify/mock"
+)
+
+type MockSQS struct {
+	mock.Mock
+}
+
+type SQSAPI interface {
```

Comment:
Is this interface definition needed? Elsewhere the `sqsconsumer.SQSAPI` interface is used instead.